pkg/kube/fake_pod_controller.go

```diff
@@ -0,0 +1,110 @@
+package kube
+
+import (
+	"context"
+	"io"
+	"time"
+
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+)
+
+type FakeKubePodController struct {
```

Comment:
Do we need `Kube` in the name? The package is already named `kube`. It would be redundant when invoking from outside the package. `kube.FakePodController` seems better than `kube.FakeKubePodController`.

Same suggestion for other objects in the file

Reply:
done